.github/pull_request_template.md

```diff
@@ -6,12 +6,15 @@ Please write a few sentences describing the overall goals of the pull request's
 
 ## PR checklist
 
-Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")
-
-- [ ] **changelog** provided, or not required
-- [ ] **3.6 backport** done, or not required
-- [ ] **2.28 backport** done, or not required
-- [ ] **tests** provided, or not required
+Please remove the segment/s on either side of the | symbol as appropriate, and add any relevant link/s to the end of the line.
+If the provided content is part of the present PR remove the # symbol.
+
+- [ ] **changelog** provided | not required because: 
+- [ ] **development PR** provided # | not required because: 
+- [ ] **framework PR** provided # | not required because: 
```

Comment:
The framework link will be to another repo. We don't want people to just copy the number: that would create a link in this repo.
```suggestion
- [ ] **framework PR** provided Mbed-TLS/mbedtls-framework# | not required because: 
```